test/integration/scheduler/queue_test.go

```diff
@@ -507,6 +507,39 @@ func TestCoreResourceEnqueue(t *testing.T) {
 			wantRequeuedPods:          sets.New("pod2"),
 			enableSchedulingQueueHint: []bool{true},
 		},
+		{
+			name:         "Pod rejected by the NodeUnschedulable plugin is requeued when the node is turned to ready",
```

Comment:
Could you also add a test case where creating unschedulable node won't requeue the pod?

Reply:
Thank you for reviewing.
I added the third test case.